src/components/dialogs/dialogs.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
- If you please rename this file to `index.js`, we'll be able to import it with its directory: `src/components/dialogs`.
 - Also this can be in a separated PR as it's a shared component.